mongodb/README.md

```diff
@@ -14,25 +14,27 @@ The job template requires a number of parameters to be specified. These can be v
 ```
 $ oc process --parameters -f mongodb-backup-s3-job-template.yaml
 NAME                              DESCRIPTION                                                           GENERATOR           VALUE
-AWS_ACCESS_KEY_ID                 AWS Access Key ID
-AWS_SECRET_ACCESS_KEY             AWS Secret Access Key
-AWS_S3_BUCKET_NAME                Name of an existing Amazon S3 bucket where backups are to be pushed
-MONGODB_HOST                      MongoDB host to target
+AWS_ACCESS_KEY_ID                 AWS Access Key ID                                                                         
+AWS_SECRET_ACCESS_KEY             AWS Secret Access Key                                                                     
+AWS_S3_BUCKET_NAME                Name of an existing Amazon S3 bucket where backups are to be pushed                       
+MONGODB_HOST                      MongoDB host to target                                                                    
 MONGODB_PORT                      MongoDB port number                                                                       27017
-MONGODB_USER                      MongoDB user to perform the backup
-MONGODB_PASSWORD                  MongoDB user password
+MONGODB_USER                      MongoDB user to perform the backup                                                        
+MONGODB_PASSWORD                  MongoDB user password                                                                     
 MONGODB_AUTHENTICATION_DATABASE   MongoDB database to authenticate against                                                  admin
-GPG_RECIPIENT                     GPG recpient name to be used to encrypt the database archive
-GPG_PUBLIC_KEY                    GPG public key content (base64 encoded)
-GPG_TRUST_MODEL                   GPG encryption trust model, defaults to "always"
+GPG_RECIPIENT                     GPG recpient name to be used to encrypt the database archive                              
+GPG_PUBLIC_KEY                    GPG public key content (base64 encoded)                                                   
+GPG_PRIVATE_KEY                   GPG private key content (base64 encoded)                                                  
+GPG_TRUST_MODEL                   GPG encryption trust model, defaults to "always"                                          always
+GPG_PASSWORD                      GPG password to be used when signing the backup archive                                   
```

Comment:
@odra AFAIK to encrypt a file using GPG we shouldn't need the password or private key. These should only be required when decrypting the file? The public key is all that's required when encrypting.

Example:
```
gpg -r <recipient-email> --yes --trust-model always --encrypt <filename>
```